app/main.py

```diff
@@ -1,6 +1,22 @@
+from functools import wraps
 from typing import Callable
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    @wraps(func)
+    def wrapper(*args, **kwargs) -> None:
```

Comment:
`wrapper` should return `Any`, not only `None`. Change your return type annotation, please.